src/group_theory/perm/cycle_type.lean

```diff
@@ -205,6 +222,46 @@ begin
   rw [←card_cycle_type_eq_one, hn, card_repeat, h2],
 end
 
+theorem is_conj_of_cycle_type_eq {σ : perm α} :
+  ∀ {τ : perm α}, cycle_type σ = cycle_type τ → is_conj σ τ :=
```

Comment:
What is the reason for having `τ` after the colon?

Reply:
I'd have to revert it before doing `cycle_induction_on _ σ`.

Reply:
In my opinion, it's better to have both `τ` and `cycle_type σ = cycle_type τ` before the colon, despite having to revert them for induction (https://leanprover-community.github.io/contribute/style.html#hypotheses-left-of-colon).

Reply:
Done, actually took the same number of lines...